### ISSUE TITLE
phantom-testkit deprection

### ISSUE THREAD

**Issue by AvihayTsayeg** [outworkers/phantom#526]
Hi,
Why this project become detracted (phantom-testkit) ?
Is there any replacement for this project for Scala Cassandra projects ?
Does phantom going to support embedded test in other way ?

Regards, Avihay

**Comment by alexflav23** [comment]
Hi @AvihayTsayeg,

Yes. There are very good reasons why we deprecated it, as we needed to separate the embedded support to a different plugin, namely `phantom-sbt`. This is because you cannot benefit from an embedded Cassandra and accurately control its lifecycle unless it lives outside of tests.

Read more here [http://outworkers.com/blog/post/phantom-tips-tip2-testing-with-phantom-sbt](http://outworkers.com/blog/post/phantom-tips-tip2-testing-with-phantom-sbt).

Regards,
